src/sentry/db/postgres/base.py

```diff
@@ -47,6 +47,14 @@ def remove_surrogates(value):
 
 
 def clean_bad_params(params):
+    # Support dictionary of parameters for %(key)s placeholders
+    # in raw SQL queries.
+    if isinstance(params, dict):
+        for key, param in params.items():
+            if isinstance(param, ((str,), bytes)):
```

Comment:
Should this be a tuple of `str, bytes`? It looks like there  might be a nested tuple here.